src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-password-form/user-password-form.component.ts

```diff
@@ -61,6 +80,13 @@ export class UserPasswordFormComponent {
     );
   }
 
+  checkPassword() {
+    const password = this.userForm.get('newpassword').value;
+    const passwordStrength = this.userChangePasswordService.checkPasswordComplexity(password);
```

Comment:
```suggestion
    const passwordStrength: [
      string,
      string
    ] = this.userChangePasswordService.checkPasswordComplexity(password);
```
or better use
```suggestion
    [this.passwordStrengthLevel, this.passwordStrengthDescription] = this.userChangePasswordService.checkPasswordComplexity(password);
```
Don't forget to remove the following lines then.
```
    this.passwordStrengthLevel = passwordStrength[0];
    this.passwordStrengthDescription = passwordStrength[1];
```